enterprise/cypher/compatibility-spec-suite/src/test/resources/blacklists/default.txt

```diff
@@ -9,3 +9,37 @@ In-query call to procedure with argument of type INTEGER accepts value of type F
 // TODO tck enforces specific order of `keys`, remove from blacklist when tck has been updated.
 Copying properties from literal map with ON CREATE
 Copying properties from literal map with ON MATCH
+
+// TCK side effects are incorrectly specified -- need update
```

Comment:
OK. I'm starting to understand. All blacklists have these same lines, so all these tests need their side effects corrected. I guess that is followup work?

Reply:
I also notice no blacklist for 3.2, so it seems 3.3 TCK is missing support for testing against 3.2? Perhaps a separate PR for that would be good.

Reply:
Good point! That configuration seems to be missing, indeed. It is out of scope for this PR though.